_includes/disqus_comments.html

```diff
@@ -2,11 +2,6 @@
 
   <div id="disqus_thread"></div>
   <script>
-    var disqus_config = function () {
-      this.page.url = '{{ page.url | absolute_url }}';
-      this.page.identifier = '{{ page.url | absolute_url }}';
-    };
-
```

Comment:
These lines are from official Disqus. I think that `page.url` may have some errors. Maybe you should check what errors reported in the console.